packages/client/src/actions/follow.ts

```diff
@@ -0,0 +1,51 @@
+import type {
+  CreateFollowRequest,
+  CreateUnfollowRequest,
+  FollowResult,
+  UnfollowResult,
+} from '@lens-protocol/graphql';
+import { FollowMutation, UnfollowMutation } from '@lens-protocol/graphql';
+import type { ResultAsync } from '@lens-protocol/types';
+
+import type { SessionClient } from '../clients';
+import type { UnauthenticatedError, UnexpectedError } from '../errors';
+
+/**
+ * Follow an Account
+ *
+ * ```ts
+ * const result = await follow(sessionClient,
+ *  { account: EvmAddress('0x90c8c68d0Abfb40D4fCD72316A65e42161520BC3') }
```

Comment:
```suggestion
 *  { account: evmAddress('0x90c8c68d0Abfb40D4fCD72316A65e42161520BC3') }
```

it's an helper function.

Reply:
Formatting it's a bit odd, is your IDE formatting these for you?